test_project/tests/integration_tests/test_admin/test_export.py

```diff
@@ -49,3 +52,123 @@ def test_export_using_admin_model(client: Client, superuser: User):
     assert ExportJob.objects.exists()
     export_job = ExportJob.objects.first()
     assert export_job.export_status == ExportJob.ExportStatus.EXPORTED
+
+
+@pytest.mark.django_db(transaction=True)
+def test_export_progress_for_sync_mode(
+    client: Client,
+    superuser: User,
+):
+    """Test export job admin progress page."""
+    client.force_login(superuser)
+
+    artist_export_job = ArtistExportJobFactory()
+    artist_export_job.refresh_from_db()
+
+    response = client.post(
+        path=reverse(
+            "admin:export_job_progress",
+            kwargs={"job_id": artist_export_job.pk},
+        ),
+    )
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json()["status"] == artist_export_job.export_status.title()
+
+
+@pytest.mark.django_db(transaction=True)
+def test_export_progress_for_async_mode(
+    client: Client,
+    superuser: User,
+    mocker: pytest_mock.MockerFixture,
+):
+    """Test export job admin progress page."""
+    client.force_login(superuser)
+
+    # Prepare data to imitate intermediate task state
+    mocker.patch(
+        "celery.result.AsyncResult.info",
+        new={"current": 2, "total": 3},
+    )
+    expected_percent = 66
+    artist_export_job = ArtistExportJobFactory()
+    artist_export_job.export_status = ExportJob.ExportStatus.EXPORTING
+    artist_export_job.save()
+
+    response = client.post(
+        path=reverse(
+            "admin:export_job_progress",
+            kwargs={"job_id": artist_export_job.pk},
+        ),
+    )
+    assert response.status_code == status.HTTP_200_OK
+    json_data = response.json()
+    assert "total" in json_data
+    assert "current" in json_data
+    assert response.json()["percent"] == expected_percent
```

Comment:
Why `response.json()` again?

Maybe prepare an object like `{"percent": expected_percent, **fake_progress_info}` and compare with it?